engine.py

```diff
@@ -45,11 +46,17 @@ class SceneEventWatcher(object):
     Specifying run_once=True in the constructor causes all events to be
     cleaned up after the first one has triggered
     """
-    def __init__(self, cb_fn,
-                 scene_events = [OpenMaya.MSceneMessage.kAfterOpen,
-                                 OpenMaya.MSceneMessage.kAfterSave,
-                                 OpenMaya.MSceneMessage.kAfterNew],
-                 run_once=False):
+
+    def __init__(
+        self,
+        cb_fn,
+        scene_events=[
```

Comment:
Since this is working and it's not a new change, I'm going to leave it.
However, we could I guess put `scene_events=None` and then define it in the `__init__` if it is `None` if we think that is better?